client/app/components/HomePageTemplate/index.js

```diff
@@ -56,99 +56,74 @@ const HomePageTemplate = ({ data }) => (
             conflicting.... To provide you with the fastest & most accurate Data
             Hygiene services and the best customer service in the industry.
           </h4>
-          <ScrollToButton fadeIn scrollTo="#contactForm" text="Contact Us" />
+          {/* <ScrollToButton fadeIn scrollTo="#contactForm" text="Contact Us" /> */}
         </div>
       </div>
     </section>
     <section
       className="section"
       style={{ padding: '0', backgroundColor: 'whitesmoke' }}
     >
-      <div className="container">
-        <div className={`columns is-multiline is-centered ${styles.textCenter}`}>
-          <div
-            className="column is-2"
-            style={{
-              border: '2px solid #ddd',
-              fontSize: '1.2rem',
-              backgroundColor: '#eee',
-            }}
-          >
-            <TimeCounter initialValue={120938} interval={1000} increment={1} />
-            <br />
-            Happy Customers
+    <div className="container">
+      <nav className="level">
```

Comment:
@leseanbiz FYI excellent use of the Bulma Level component, this is exactly the right use-case for that css and it looks sooo much better than the original design. Good job :+1:

Reply:
Thanks! I did something right! :)